api/components/docs/changeLog/FIS12_2.1.0_010824.md

```diff
@@ -0,0 +1,13 @@
+### Change Logs:  Invoice Based Loans API specs
+
+| Version                         | FIS12_2.1.0_010824 |
+| :------------------------------ | :----------------- |
+| Updates in API Specs on dev doc | 1st August 2024      |
+
+##### Invoice Based Loans:
+
+- Added the missing KFS terms required for offer generation.
```

Comment:
Also Add the change of making AA optional

Reply:
done